onboarding_documentation/technical_documentation/metamist_docker/team_docs_metamist_tutorial.py

```diff
@@ -0,0 +1,160 @@
+import os
```

Comment:
I've left comments here instead of the tutorial, so ensure any changes here get reflected above.